arbitrum-docs/how-arbitrum-works/04-geth-at-the-core.mdx

```diff
@@ -17,12 +144,14 @@ Please note any links on this page may be referencing old releases of Nitro or o
 
 :::
 
-## Hooks
+### Hooks
 
 Arbitrum uses various hooks to modify Geth's behavior when processing transactions. Each provides an opportunity for ArbOS to update its state and make decisions about the transaction during its lifetime. Transactions are applied using Geth's [`ApplyTransaction`][applytransaction_link] function.
 
 Below is [`ApplyTransaction`][applytransaction_link]'s callgraph, with additional info on where the various Arbitrum-specific hooks are inserted. Click on any to go to their section. By default, these hooks do nothing so as to leave Geth's default behavior unchanged, but for chains configured with [`EnableArbOS`](#enablearbos) set to true, [`ReadyEVMForL2`](#ReadyEVMForL2) installs the alternative L2 hooks.
 
+> > > > > > > master:arbitrum-docs/how-arbitrum-works/arbos/geth.mdx
```

Comment:
Missing diagram here

Reply:
Looking at published doc, it seems the flow between "installs the alternative L2 hooks" goes right into the "diagram" below.